packages/client/src/core/type/util.test.ts

```diff
@@ -190,7 +191,7 @@ describe('util', () => {
       document.head.innerHTML = `
       <link rel="shortcut icon" sizes="16x16 24x24" href="/favicon.ico">
     `;
-      expect(getFavicon()).toEqual('http://localhost/favicon.ico');
+      expect(getFavicon()).toEqual('http://localhost:3000/favicon.ico');
```

Comment:
Not entirely sure if this is acceptable, but in the vitest environment window.location.host includes the port whereas in the jest environment it does not